src/jsd/actuator.cc

```diff
@@ -690,9 +665,15 @@ std::string fastcat::Actuator::GetFastcatFaultCodeAsString(
 {
   std::string fault_str;
 
-  if (state.type == ACTUATOR_STATE) {
-    auto fault = static_cast<ActuatorFastcatFault>(
-        state.actuator_state.fastcat_fault_code);
+  if (state.type == EGD_ACTUATOR_STATE || state.type == EPD_ACTUATOR_STATE) {
+    ActuatorFastcatFault fault;
+    if (state.type == EGD_ACTUATOR_STATE) {
+      fault = static_cast<ActuatorFastcatFault>(
+          state.egd_actuator_state.fastcat_fault_code);
+    } else {
+      fault = static_cast<ActuatorFastcatFault>(
+          state.epd_actuator_state.fastcat_fault_code);
+    }
```

Comment:
These static functions are needed for client applications that do not have access to an instance of the driver, only instances of the state. For example, c0_mech.
We need to check for type because we now have two different actuator states and we do not use inheritance in them. I tried to produce the best possible outcome with how things are set up right now.